csm_web/frontend/static/frontend/css/style.css

```diff
@@ -35,15 +36,13 @@ button {
 	--csm-green: #9ae0b3;
 	--csm-green-darkened: #75b38a;
 	--csm-green-lightened: #d7ffe5;
-	--csm-theme-cs61a: var(--csm-green);
-	--csm-theme-cs61b: #63c5ff;
-	--csm-theme-cs61c: #797c99;
-	--csm-theme-cs70: #ff7070;
-	--csm-theme-cs88: #e46495;
-	--csm-theme-ee16a: #707eff;
-	--csm-theme-eecs16a: var(--csm-theme-ee16a);
-	--csm-theme-ee16b: #ffe070;
-	--csm-theme-eecs16b: var(--csm-theme-ee16b);
+	--csm-theme-csm61a: var(--csm-green);
+	--csm-theme-csm61b: #63c5ff;
+	--csm-theme-csm61c: #797c99;
+	--csm-theme-csm70: #ff7070;
+	--csm-theme-csm88: #e46495;
+	--csm-theme-csm16a: #707eff;
+	--csm-theme-csm16b: #ffe070;
```

Comment:
I think this refactor kinda messed up a lot of the colors on the website. this is what hows up on my screen, and i suspect it has something to do with this refactor? 
<img width="827" alt="Screen Shot 2023-01-02 at 7 10 55 AM" src="https://user-images.githubusercontent.com/62233299/210199505-01f5e1c5-984d-4c67-8f0a-a4e7287b1a42.png">

Reply:
oh is this intended behavior where you want the official cs61 courses to not have any color yet ig?

Reply:
Yeah we want the restricted courses to look distinct from the CSM courses, and since scheduler is a part of CSM, leaving the CSM colors intact while making all restricted courses black I think is a good compromise. This means that going forward, all CSM courses on scheduler will be named "CSMxxx" and the normal courses will be named normally.

I do still welcome any suggestions for making the course cards look distinct but still colorful though.

Reply:
i think it'll be a fun frontend project maybe for this semester! we can leave it up in the air?